src/components/Edit/DocumentsDelivery.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <div class="pb-6" id="document-delivery-section">
-    <h2>{{sectionNumber}} Alteration Documents Delivery</h2>
-    <div class="pt-4 pb-4">Copies of the alteration documents will be sent
+    <h2>{{sectionNumber}} Documents Delivery</h2>
+    <div class="document-info pt-4 pb-4">Copies of the {{ getFilingName.toLowerCase() }} documents will be sent
       to the following email address listed below.</div>
     <div :class="{ 'invalid-section': documentDeliveryInvalid }">
-      <v-card flat class="pt-4 pr-8">
+      <v-card flat class="section-container">
         <v-container>
-          <v-row class="pl-4">
+          <v-row>
             <v-col cols="3" class="px-0">
```

Comment:
You might want to add `cols="12"` and `md="3"` to the cols to promote wrapping below size "md".

Reply:
If we are aiming for responsive we will need to update an awful lot. 
Were these additional requirements in Registrations?

Reply:
No, they were not requirements.

I updated various rows/cols in Create UI because I noticed they were not acting similar on the review page (and it's good practise 😁 ).

You can ignore this conversation.